inference-engine/tests/functional/plugin/cpu/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -20,6 +20,9 @@ std::vector<std::string> disabledTestPatterns() {
         // TODO: Issue 32023
         R"(.*(QuantGroupConvBackpropData2D)*QG=Pertensor.*)",
         // TODO: Issue 31845
-        R"(.*(FakeQuantize).*)"
+        R"(.*(FakeQuantize).*)",
+        // TODO: Issue: 32521
+        R"(.*(EltwiseLayerTest).*secondaryInputType=CONSTANT.*netPRC=FP16.*)",
```

Comment:
be precise here, i am pretty sure that many of the constant FP16 tests are passing

Reply:
Additionally please rebase, supposedly fix was merged it would be good to confirm it now and remove this pattern early